src/test/java/org/folio/rest/api/TestBaseWithInventoryUtil.java

```diff
@@ -217,7 +218,7 @@ protected static Item buildItem(UUID holdingsRecordId,
     itemToCreate.put("id", UUID.randomUUID().toString());
     itemToCreate.put("holdingsRecordId", holdingsRecordId.toString());
     itemToCreate.put("barcode", Long.toString(new Random().nextLong()));
-    itemToCreate.put("status", new JsonObject().put("name", "Available"));
+    itemToCreate.put("status", new JsonObject().put("name", Status.Name.AVAILABLE.value()));
```

Comment:
Is that a generated constant from the production code?

Reply:
Yes, it is enum value generated by RMB.

Reply:
I think that API tests should be minimally coupled to the details of the production code. Both in order to reduce the chance of false diagnostics (e.g. passing or failing for the wrong reason) and to improve maintainability by making the less brittle to internal design changes.

I can understand the desire to use a constant value. I think that constant should be defined in the test context not taken from the production code.

What do you think?

Reply:
I agree, there might be a place for false passing. Changed

Reply:
Please can the mod-inventory pull request be updated similarly, if you haven't already done so